doc/man3/EVP_desx_cbc.pod

```diff
@@ -23,6 +23,10 @@ All modes below use a key length of 128 bits and acts on blocks of 128-bits.
 
 The DES-X algorithm in CBC mode.
 
+This algorithm is not provided by the default OpenSSL provider.
+To use them it is necessary to load either legacy OpenSSL provider or another
```

Comment:
d/them/

_the OpenSSL legacy provider_ instead of _legacy OpenSLL provider_.

Reply:
Done. Thanks!